src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterSubsystem extends SubsystemBase {
+  private VictorSP shooterMotorOne, shooterMotorTwo, shooterMotorThree;
+  
+  /**
+   * Creates a new ShooterSubsytem.
+   */
+  public ShooterSubsystem() {
+    shooterMotorOne = new VictorSP(7);
```

Comment:
instead of hard-coding ports, use the ports in Constants.java (each motor must be a different port)